docs/changelog.rst

```diff
@@ -10,20 +10,17 @@ Breaking changes
 This release makes the following changes to the API.  You may need to update your implementation to
 accommodate these changes.
 
+- Add support for multiple resources in ModelAdmin. (#1223)
```

Comment:
Just curious about why are these changelog entries here?

Reply:
These entries are merged in from the current release-3-x branch - ideally they shouldn't be listed yet, but it will all work out when merged.